nest_elephant_tvb/translation/RichEndPoint.py

```diff
@@ -0,0 +1,202 @@
+# Copyright 2020 Forschungszentrum Jülich GmbH and Aix-Marseille Université
+# "Licensed to the Apache Software Foundation (ASF) under one or more contributor license agreements; and to You under the Apache License, Version 2.0. "
+from mpi4py import MPI
+import pathlib
+
+def make_connections(path_to_files_receive, path_to_files_send, logger_master):
+    '''
+    Rich End Point. Still a first draft, will be changed to a proper interface.
+    This function establishes two MPI intercommunicators. One to NEST and one to TVB.
+    :param path_to_files_receive: path to file, store information about receiving MPI connection
+    :param path_to_files_send: path to file, store information about sending MPI connection
+    :param logger_master: main logger for the connections
+    '''
+    use_mpi = True
+    # init MPI
+    if use_mpi:
+        comm = MPI.COMM_WORLD  # INTRA communicator
+        info = MPI.INFO_NULL
+        root=0
+    else:
+        comm = None
+    ### nest connection
+    comm_receiver, port_receive = nest_connection(comm, root, info, logger_master, path_to_files_receive)
+    ### tvb connecction
+    comm_sender, port_send = tvb_connection(comm, root, info, logger_master, path_to_files_send)
+    logger_master.info('Connections made, starting translation...')
+    
+    return comm, comm_receiver, port_receive, comm_sender, port_send
+
+
+def nest_connection(comm, root, info, logger_master, path_to_files_receive):
```

Comment:
This will be revisited with the next refactor step (here: proper implementation of the end point / interface).
Added this to [Issue 3](https://github.com/multiscale-cosim/TVB-NEST/issues/3)